src/app/helpers/whitelistApis.js

```diff
@@ -1626,7 +1626,55 @@ const API_LIST = {
     '/dataset/v1/request/submit': {
       checksNeeded: ['ROLE_CHECK'],
       ROLE_CHECK: [ROLE.ORG_ADMIN, ROLE.CONTENT_CREATOR, ROLE.COURSE_MENTOR]
-    }
+    },
+    '/kendra/v1/users/entityTypesByLocationAndRole/:stateId': {
+      checksNeeded: ['ROLE_CHECK'],
+      ROLE_CHECK: [ROLE.PUBLIC]
```

Comment:
please do not give public role specify which role can access this URL making it public will break the purpose why the whitelisting was built